.github/workflows/w3up-client.yml

```diff
@@ -31,8 +31,6 @@ jobs:
         uses: actions/checkout@v3
       - name: Install
         uses: pnpm/action-setup@v4
-        with:
-          version: 9
```

Comment:
I've removed the version because it was causing conflicts with the version defined in the root package.json. It is optional when you have it in package.json: https://github.com/pnpm/action-setup?tab=readme-ov-file#version